contracts/JBProjects.sol

```diff
@@ -160,4 +160,17 @@ contract JBProjects is IJBProjects, JBOperatable, ERC721Votes, Ownable {
 
     emit SetTokenUriResolver(_newResolver, msg.sender);
   }
+
+  /**
+    @dev See {IERC165-supportsInterface}.
+  */
+  function supportsInterface(bytes4 interfaceId)
```

Comment:
My thinking here was that since the contract inherits (I)ERC721 it already contains ERC165 but it doesn't expose the support for the IJBProjects interface, but I'm not sure what it would be used for. 

Should we keep it or remove it?

Reply:
that makes sense! might as well keep it and extend.